Data/Sys/GameSettings/RM3.ini

```diff
@@ -19,6 +19,7 @@ EmulationIssues =
 
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
+UseRealXFB = False # Prevent black bar
```

Comment:
Which one do you want to enforce? Virtual XFB? Or just to prohibit real XFB? I doubt real XFB could be broken while virtual XFB works fine....